plugins/modules/monitoring/datadog/datadog_event.py

```diff
@@ -113,6 +131,7 @@ def main():
         argument_spec=dict(
             api_key=dict(required=True, no_log=True),
             app_key=dict(required=True, no_log=True),
+            api_host=dict(type='str', required=False),
```

Comment:
Not that this has any effect whether `required=False` is there or not :)
```suggestion
            api_host=dict(type='str'),
```